custom_components/tahoma/__init__.py

```diff
@@ -76,8 +77,12 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
     username = entry.data.get(CONF_USERNAME)
     password = entry.data.get(CONF_PASSWORD)
 
+    session = aiohttp_client.async_create_clientsession(
```

Comment:
Can you link where did you see we need this please?

Reply:
https://developers.home-assistant.io/docs/integration_quality_scale_index/#platinum-

Reply:
Cookie is important to stay logged in and keep the state, afaik.

Reply:
You are right, I can see a JSESSIONID cookie.

Reply:
@tetienne verify_ssl is optional, I removed this one. However, the CookieJar is required apparently. Without the CookieJar, the cookie is not saved and thus the session is lost.